lib/dpl/provider/pypi.rb

```diff
@@ -65,7 +65,7 @@ def needs_key?
       def push_app
         context.shell "python setup.py register -r #{options[:server] || 'pypi'}"
         context.shell "python setup.py #{options[:distributions] || 'sdist'} upload -r #{options[:server] || 'pypi'}"
-        context.shell "python setup.py upload_docs --upload-dir #{options[:docs_dir] || 'build/docs'}"
+        context.shell "python setup.py upload_docs#{options[:docs_dir] ? ' --upload-dir ' + options[:docs_dir] : ''}"
```

Comment:
Do not embed the option logic inside `#{}`, please. This makes the logic unclear.